snooty/rstspec.toml

```diff
@@ -646,8 +646,10 @@ example = """.. banner::
 [directive."mongodb:cta-banner"]
 content_type = "block"
 options.url = { type = "uri", required = true }
+options.icon = "string"
```

Comment:
Should this be a closed set of values?

Reply:
So the frontend handles the logic and error checking for dealing with whether or not the option being passed in is a valid one. If we wanted to have a closed set of values, we'd have to include all of the potential LG icons, and there are a lot.

Reply:
For reference, there are about 88 potential icons to choose from [here](https://www.mongodb.design/component/icon/example/) (for now). LG does allow us to import the list of possible values on the frontend, which makes it easier to handle there with our current version of the LG Icon package if we don't want to actively maintain the same set in our rstspec.